docs/source/models/concept_models/vivarium_alzheimers/concept_model.rst

```diff
@@ -314,9 +314,16 @@ scenario, and input draw.
   * - Person-time in each Alzheimer's cause state
     -
     -
-  * - Test counts (CSF, PET, BBBM)
-    - By diagnosis provided (for BBBM tests)
+  * - Existing (CSF/PET) eligible person-time
```

Comment:
Is this going to be person-time or count of simulants? Because we decided on counts for the BBBM testing, right? And it seems like we should be observing comparable things.